partner_contact_person/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    contact_person_id = fields.Many2one(
+        string="Contact Person",
+        comodel_name="res.partner",
+        help="This partner is the contact person for this company",
+        domain="[('is_company','=',False)]",
```

Comment:
Could you add copy prevention? I've checked that, when a contact is duplicated, contact person is copied as well, and this is definitely wrong (a contact that doesn't belong to a company is shown as contact person of the new company)
```suggestion
        domain="[('is_company','=',False)]",
        copy=False,
```